docs/user-guide/installation/f23-.rst

```diff
@@ -10,14 +10,9 @@ Repositories
 Pulp
 ^^^^
 
-Download the appropriate repo definition file from the Pulp repository:
+Enable appropriate COPR repository, according to the instructions found on the COPR site:
```

Comment:
This could become an in-line link. Here is a markdown example, I forge the rst equivalent.
`[instructions found on the COPR site](https://copr.fedorainfracloud.org/groups/g/pulp/coprs/)`